### ISSUE TITLE
Old license in a couple of files

### ISSUE THREAD

**Issue by Enchufa2** [mpimd-csc/flexiblas#44]
I see:

```bash
$ grep -r "As a special exception"
test/usage/python-numpy-scipy/numpy-scipy-benchmark.py:# As a special exception, the copyright holders of FlexiBLAS give you permission
src/flexiblas_api.h.in: * As a special exception, the copyright holders of FlexiBLAS give you permission
```

Notably the second one.

**Comment by Enchufa2** [comment]
The testing bug was in LAPACK, so no problem. :) It would be better for me if you increment the patch version to 3.4.1, because I already built and published updates for 3.4.0.

**Comment by Enchufa2** [comment]
Thanks, I've updated the R package too to follow the same licensing.
